src/main/java/org/apache/commons/collections4/map/CaseInsensitiveMap.java

```diff
@@ -141,6 +141,70 @@ protected Object convertKey(final Object key) {
         return AbstractHashedMap.NULL;
     }
 
+    /**
```

Comment:
This javadoc duplicates that inherited from the parent so can be dropped.

Reply:
dropped.